packages/svelte/src/internal/client/runtime.js

```diff
@@ -1899,3 +1899,26 @@ if (DEV) {
 	throw_rune_error('$inspect');
 	throw_rune_error('$props');
 }
+
+/**
+ * Expects a value that was wrapped with `freeze` and makes it frozen.
+ *
```

Comment:
it looks like we generally haven't put a new line after a description elsewhere in this file
```suggestion
```